### PR TITLE
Add in new Plover outlines for "pattern".

### PR DESCRIPTION
**Pull request by paulfioravanti**
In the current `dict.json` file, there are the following strokes for "pattern", all of which are valid in Plover release [weekly-v4.0.0.dev8+66.g685bd33](https://github.com/openstenoproject/plover/releases/tag/weekly-v4.0.0.dev8%2B66.g685bd33):

```json
"PAT/ERPB": "pattern",
"PAT/TERPB": "pattern",
"PAT/TOERPB": "pattern",
```

In the `top-10000-project-gutenberg-words.json` dictionary, the stroke used for "pattern" is:

https://github.com/didoesdigital/steno-dictionaries/blob/e9408eec8e47755e728bb1d68fce29a58ba0f121/dictionaries/top-10000-project-gutenberg-words.json#L5038

`PA*RPBT` curiously isn't contained in any of the other dictionaries, but regardless, it currently outputs "PA*RPBT" as-is, so this entry should be removed.

Using Plover's search function to look up outlines for "pattern" returns:

```
pattern:
   PAERPB
   T*ERPB
   PA*ERPB
   AT/TERPB
   PAT/ERPB
   PAT/*ERPB
   PAT/TERPB
   PAT/TOERPB
```

So, this PR proposes to add outlines in this list that are not currently in the dictionaries to `dict.json`, and choose `PAERPB` as the stroke for the Gutenberg dictionary, and consequently for Typey Type.

### CODE REVIEW

File: dictionaries/dict.json [hunk 1]
```diff
@@ -48752,7 +48753,7 @@
 "PAER/SO*PB": "Pearson",
 "PAERBL": "partial",
 "PAERL": "parole",
-"PAERPB": "partner",
```
Comment (didoesdigital) [inline dictionaries/dict.json:48755]: Let's add `"PAERPB": "partner",` to bad habits

**Comment by didoesdigital** [comment]
Thanks, @paulfioravanti ! `PA*RPBT` was a personal brief of mine that snuck in.

Now that the `briefs.json` dictionary contains only briefs chunked out from the Plover dictionary instead of personal briefs (since https://github.com/didoesdigital/steno-dictionaries/issues/123 was  closed), I wonder if it's worth starting to move more examples of briefs that are in no way phonetic out of `dict.json` into `briefs.json`, such as `"T*ERPB": "pattern",`. What do you think?

File: dictionaries/dict.json (hunk 1 above)
Comment (paulfioravanti) [reply dictionaries/dict.json:48755]: Done.

**Comment by paulfioravanti** [comment]
> I wonder if it's worth starting to move more examples of briefs that are in no way phonetic out of dict.json into briefs.json, such as "T*ERPB": "pattern",. What do you think?

Sounds good to me. So, I'm guessing proposals for additions to `briefs.json` could end up being a bit subjective (an outline that sounds phonetic to one, perhaps may not sound so to another), but I'm happy to propose additions in future PRs based on gut feel, and we can discuss :)

For this PR, though, I'll move `"T*ERPB": "pattern"` into `briefs.json` (and while I'm at it, I'll sort `briefs.json` in a separate commit).

**Comment by didoesdigital** [comment]
Great, thanks @paulfioravanti and especially good thinking on sorting `briefs.json`—that will avoid some conflict headaches.

**Pull request merged**